control-plane/connect-inject/controllers/endpoints/endpoints_controller_ent_test.go

```diff
@@ -2107,7 +2111,8 @@ func TestReconcileDeleteGatewayWithNamespaces(t *testing.T) {
 
 				if tt.enableACLs {
 					_, _, err = consulClient.ACL().TokenRead(token.AccessorID, nil)
-					require.EqualError(t, err, "Unexpected response code: 403 (ACL not found)")
+					require.EqualError(t, err, 
```

Comment:
I have a suggestion to potentially make these tests less brittle for the future. 

Can we switch all of the ACL error string matches to just check for `contains` like
`require.Contains(t, err, "ACL not found")` or something like that

Reply:
Will do, the reason I didn't change them all to this assertion was to stay precise (if it actually matters to be able to distinguish between the two cases). Thanks!